src/auth/src/credentials.rs

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 pub(crate) mod mds_credential;
-pub(crate) mod service_account_credential;
+mod service_account_credential;
```

Comment:
We kept these at crate level visibility because we wanted to avoid exposing exact credential types unless required.

Is there a reason to make this change?

Reply:
Now it is private. `pub mod ...` would make it externally visible. It did not need to be `pub(crate)` because only this file needs it.

Reply:
Got it.